integrations/appium.js

```diff
@@ -0,0 +1,37 @@
+require('dotenv').config()
+
+const { Print } = require('@ianwalter/print')
+
+module.exports = class AppiumIntegration {
+  constructor (context) {
+    // Set up a print instance on the integration instance so it can be reused.
+    this.print = new Print({ level: context.logLevel })
+    this.print.debug('Appium integration enabled')
+
+    // Define the global capability options.
+    context.webdriver.port = context.webdriver.port || 4723
+  }
+
+  static integrate (context) {
+    if (context.webdriver.appium) {
+      context.webdriver.integrations.push(new AppiumIntegration(context))
+    }
+  }
+
+  enhanceCapability (testContext) {
+    const options = {
+      // Tell appium the name of the test.
+      sessionName: testContext.key
+    }
+    testContext.capability = Object.assign(options, testContext.capability)
+  }
+
+  // Not sure yet if there is anything to add for reporting
+  async report ({ webdriver, testContext }) {
+    try {
+      // DO SOMETHING
+    } catch (err) {
+      this.print.error(err)
+    }
+  }
```

Comment:
Go ahead and remove this. I don't see that Appium has any kind of centralized dashboard to report results.